### PR TITLE
docs: add security section

### PR DESCRIPTION
**Pull request by anastasialanz**
Adds a list of what we do in terms of security at Sparkbox.

### CODE REVIEW

**Comment by anastasialanz** [comment]
@asimpson Do we have this built into our workflow anywhere or is this something that we should be doing?

**Comment by asimpson** [comment]
> @asimpson Do we have this built into our workflow anywhere or is this something that we should be doing?

It's not built-in and I think it is something that's manual. However, we could probably add some kind cadence for running it.

File: security/security.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Sparkbox requires that all members of the Sparkbox organization in Github set up [two-factor authentication](https://github.com/sparkbox/standard/blob/master/security/two-factor-authentication.md) and use a tool like [Authy](https://authy.com/) or the built-in [two-factor authentication support in 1Password](https://support.1password.com/one-time-passwords/) instead of using an SMS number.
+
+## Soon Expired Bot
+
+[Soon Expired](https://github.com/sparkbox/soonexpired) is a Slack integration we created to monitor domains for expired SSL certificates. SSL certificiates are important for creating a secure connection between the browser and the web server.
+
+## VPN
+
+Sparkbox uses DataYard for their VPN so that team members can connect from anywhere and have their connection encrypted. Being connected to the VPN also allows us to connect to other client's internal network resources.
```
Comment (cromwellryan) [inline security/security.md]: Maybe _Sparkbox **partners with** [DataYard](https://datayard.us)_?

**Comment by cromwellryan** [comment]
I like `npm audit` as well.

What about adding [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project)? This is something we don't do regularly, but we have now used it on 2 projects. As we focus on security more in Q1, this will likely show up a lot more. We could wait until then too if you rather.

**Comment by cromwellryan** [comment]
Per my convo with @nrleak, I'm questioning where this belongs. This is currently on an island, though it feels like it should be a peer of the **[Instructions section of Security readme](https://github.com/sparkbox/standard/blob/master/security/README.md#instructions)**

cc @emilykggray do you have any thoughts about the content structure/placement?

**Comment by emilykggray** [comment]
So you think it should be consumed into the main Readme for the Security section? I think the new organization would put Security under "Development Process" and perhaps it then includes more content? Would you like all three of us to ponder and have a quick chat, @cromwellryan?

**Comment by anastasialanz** [comment]
@cromwellryan and @emilykggray, did you decide where this content should live?

**Comment by emilykggray** [comment]
I'd suggest we go ahead and merge when the content is ready, and we'll move it to a solid place when we move into the new architecture.

**Pull request closed without merging**